src/main/java/org/broadinstitute/hellbender/utils/smithwaterman/SmithWatermanAligner.java

```diff
@@ -28,6 +28,11 @@
      */
     SmithWatermanAlignment align(final byte[] ref, final byte[] alt, SWParameters parameters, SWOverhangStrategy overhangStrategy);
 
+    //methods for printing out number of SW/non-SW alignments
+    int getNumOfAlignments();
```

Comment:
In my opinion it would be better not to include these methods, for a few reasons:

*  Forcing the native implementations to have these methods requires `System.out.printLn` side effects and a placeholder `return 0`, both of which are undesirable.
*  It adds extra code to the `SmithWatermanAligner` for tracking counts.  It's not much, but one should be ruthless about any burden on developers' short-term memory.
*  It's preferable not to put into code something that can be done just as well in the debugger or profiler.